controllers/admin/AdminImportController.php

```diff
@@ -2518,7 +2518,7 @@ protected function attributeImportOne($info, $default_language, &$groups, &$attr
                             );
                         }
                         if ($field_error !== true || $lang_field_error !== true) {
-                            $this->errors[] = ($field_error !== true ? $field_error : '') . (isset($lang_field_error) && $lang_field_error !== true ? $lang_field_error : '') . mysql_error();
+                            $this->errors[] = ($field_error !== true ? $field_error : '') . (isset($lang_field_error) && $lang_field_error !== true ? $lang_field_error : '') . mysqli_error();
```

Comment:
We need to detect if we are using mysqli or PDO instead

Reply:
Like this?